src/agent/onefuzz/src/expand.rs

```diff
@@ -108,9 +110,29 @@ impl<'a> Expand<'a> {
             PlaceHolder::InputFileSha256.get_string(),
             ExpandedValue::Mapping(Box::new(Expand::input_file_sha256)),
         );
+        values.insert(
+            PlaceHolder::MachineId.get_string(),
+            ExpandedValue::Mapping(Box::new(Expand::machine_id)),
+        );
+
         Self { values }
     }
 
+    // by making this a Mapping rather than when creating an Expand instance,
+    // users only pay for fetching the machine_id when it's used in expansion
+    fn machine_id(&self, _format_str: &str) -> Option<ExpandedValue<'a>> {
+        let rt = tokio::runtime::Runtime::new().unwrap();
```

Comment:
When might this panic?

Reply:
This will panic if we can't allocate a new tokio runtime.  From my brief investigation, this should only happen in OOM situations, where we'll panic all over the place as is.

Reply:
I think this is comment-worthy (and the motivation: the existing machine ID-getting code uses async IO, and there's not a really a good way to factor out a blocking version).